pkgs/by-name/ku/kubectl-rook-ceph/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
`buildGoModule` now supports single-arg lambdas, like `mkDerivation` and `buildRustPackage`